csaf_2.0/submit/README.md

```diff
@@ -0,0 +1,84 @@
+# Submission request to advance CSAF v2.0 to an International Standard
+
+Date: January 28, 2023
+
+>*The following text parts - marked up in a gray, cursive,, and smaller font - have been copied directly from*
+>*the requirements stated in the relevant section of the current OASIS Liaison Policies page at:*
+>                                          https://www.oasis-open.org/policies-guidelines/liaison/#submitwork
+
+*Any submission request delivered to the OASIS President under this policy must be in writing, and must include the following:*
+
+a. *The name(s) of the submission requester(s), that is, the TC, the Member Section, or the OASIS*
+   *Organizational Members that support the submission request as described in section 1(d).*
+
+OASIS Common Security Advisory Framework (CSAF) TC
+
+b. *The name of the intended receiving standards organization. The request may also suggest the committee*
+   *or group in that organization which should process that submission.*
+
+ISO/IEC\ JTC\ 1 Secretariat (suggesting “SC\ 27 Information security, cybersecurity and privacy
```

Comment:
Backslashs seem to be unnecessary (looking at the rendered md file on Github). However, the pdf seems to be fine :thinking: 
```suggestion
ISO/IEC JTC 1 Secretariat (suggesting “SC 27 Information security, cybersecurity and privacy

Reply:
Please let us keep the protected space injected as a latex code. We do not want JTC<linebreak>1 etc. I am not a markdown purist esp. with this document ... it is the PDF that counts.